<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoveFile</class>
 <widget class="QWidget" name="MoveFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Основной стиль в стиле 7-Zip */
QWidget {
    background-color: #f0f0f0;
    color: #000000;
    font-family: &quot;Segoe UI&quot;, Tahoma, sans-serif;
    font-size: 9pt;
}

/* Стиль для кнопок */
QPushButton {
    background-color: #e1e1e1;
    border: 1px solid #a0a0a0;
    border-radius: 3px;
    padding: 4px 12px;
    font-weight: normal;
    min-width: 60px;
    min-height: 22px;
}

QPushButton:hover {
    background-color: #e8e8e8;
    border: 1px solid #909090;
}

QPushButton:pressed {
    background-color: #d0d0d0;
    border: 1px solid #808080;
}

QPushButton:disabled {
    background-color: #f5f5f5;
    color: #a0a0a0;
    border: 1px solid #d0d0d0;
}

/* Кнопка по умолчанию (акцентная) */
QPushButton[default=&quot;true&quot;] {
    background-color: #0078d4;
    color: white;
    border: 1px solid #005a9e;
}

QPushButton[default=&quot;true&quot;]:hover {
    background-color: #106ebe;
    border: 1px solid #004578;
}

QPushButton[default=&quot;true&quot;]:pressed {
    background-color: #005a9e;
    border: 1px solid #003966;
}

/* Стиль для ListView */
QListView {
    background-color: white;
    border: 1px solid #a0a0a0;
    border-radius: 2px;
    outline: 0;
    padding: 1px;
}

QListView::item {
    background-color: white;
    border: none;
    padding: 2px 4px;
    margin: 1px;
}

QListView::item:hover {
    background-color: #e5f3ff;
}

QListView::item:selected {
    background-color: #0078d4;
    color: white;
}

QListView::item:selected:!active {
    background-color: #e1e1e1;
    color: black;
}

/* Полоса прокрутки */
QScrollBar:vertical {
    background-color: #f0f0f0;
    width: 15px;
    margin: 0px;
}

QScrollBar::handle:vertical {
    background-color: #c0c0c0;
    border-radius: 7px;
    min-height: 20px;
    margin: 2px;
}

QScrollBar::handle:vertical:hover {
    background-color: #a0a0a0;
}

QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
    height: 0px;
}

QScrollBar:horizontal {
    background-color: #f0f0f0;
    height: 15px;
    margin: 0px;
}

QScrollBar::handle:horizontal {
    background-color: #c0c0c0;
    border-radius: 7px;
    min-width: 20px;
    margin: 2px;
}

QScrollBar::handle:horizontal:hover {
    background-color: #a0a0a0;
}

QScrollBar::add-line:horizontal, QScrollBar::sub-line:horizontal {
    width: 0px;
}

/* Дополнительные стили для окон и панелей */
QMainWindow {
    background-color: #f0f0f0;
}

QDialog {
    background-color: #f0f0f0;
}

QGroupBox {
    border: 1px solid #c0c0c0;
    border-radius: 3px;
    margin-top: 1ex;
    padding-top: 10px;
    font-weight: bold;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 10px;
    padding: 0 5px;
}</string>
  </property>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>271</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="listView_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>40</y>
     <width>281</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>От куда</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>10</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Куда</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>370</y>
     <width>511</width>
     <height>112</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btnMoveAll">
      <property name="text">
       <string>Переместить текущую директорию</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnMoveSelect">
      <property name="text">
       <string>Переместить выделенные элементы</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnQuit">
      <property name="text">
       <string>Выйти</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
